Update vpn-gateway-howto-point-to-site-resource-manager-portal.md

tylerwel opened the pull request:
Please see proposed changes. It took me me a long time to realize that I need to create an additional address space, and other people have also seen this issue.

https://social.msdn.microsoft.com/Forums/en-US/631df416-3a02-4573-9a12-f2b4ca4ed5b3/gateway-subnet-does-not-allow-me-to-create-a-subnet-with-current-subnets-avaiable?forum=WAVirtualMachinesVirtualNetwork

azurecla commented:
Hi __@JakobGSvendsen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@JakobGSvendsen - Thanks for the contribution.

@ccompy - Could you please review Jakob's contribution and let me know if we can go ahead and merge the updates?

tysonn commented:
@ChrisCompy - please review this public repo contribution - second notice

ChrisCompy commented:
There is another PR in works vis a vi one of the editors.  That PR has changes that make the contribution unneeded.  The change we made is slightly different but has the same effect.

ChrisCompy commented:
@JakobGSvendsen  Just to be clear, I do thank you for the contribution.  You are correct in what was missing.  I had a few 'bumps' in the doc of which that was probably the most egregious.  We can go ahead and take your contribution which is helpful until the doc update comes out.

tysonn commented:
@JakobGSvendsen - The code snippet has changed significantly:

```
New-AzureRmResourceGroupDeployment
-Name <name>
-ResourceGroupName <resource-group>
-TemplateFile <ArmTemplatePath>
-storageAccountNameParameter <storage-account>
-adminUsername <admin-username>
-adminPassword (ConvertTo-SecureString -force -AsPlaintext "myPassword1!")
-sqlservername <sql-server>
-sqlsysadmin 'sa'
-sqlsysadminpwd <sa-password>
-defaultSslPfxFilePassword <webapps-ssl-cert>
-resourceProviderSslPfxFilePassword <management-ssl-cert>
-environmentDnsSuffix 'webapps.azurestack.local'
-armEndpointUri 'https://api.azurestack.local'
-resourceProviderUri 'https://management.azurestack.local'   
```

And it sounds the overall article is going to change soon. There's a merge conflict due to the code snippet differences. I'm going to close at this time - I'm sorry we aren't able to merge.

tylerwel committed:
Update vpn-gateway-howto-point-to-site-resource-manager-portal.md

https://social.msdn.microsoft.com/Forums/en-US/631df416-3a02-4573-9a12-f2b4ca4ed5b3/gateway-subnet-does-not-allow-me-to-create-a-subnet-with-current-subnets-avaiable?forum=WAVirtualMachinesVirtualNetwork
